src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/custom.py

```diff
@@ -7,6 +7,28 @@
 
 # pylint: disable=too-many-locals, unused-argument, too-many-statements
 
+def create_custom_image(client, resource_group, lab_name, name, author=None, description=None,
+						source_vm_id=None, os_type=None, os_state=None):
```

Comment:
If you break the line, please align the first parameter on the second line with the first parameter of the previous line.